server/server.js

```diff
@@ -283,9 +335,14 @@ io.on('connection', (socket) => {
                     lobby.correctGuesses <
                     numberOfPlayers - 1 - lobby.correctGuesses
                 ) {
+                    lobby.pointMap.set(
```

Comment:
Ezek az if-else branchek eltérnek valami másban is, mint a konstant értékében?
Ha nem és ez a jövőben is sem várható, akkor érdemesebb felül beállítani a jutalom mértékét egy konstansba és aztán ezt egyszer hozzáadni az adatbázisban.
Szóval nem 
```ts
let score;
if (...) { score = 35; }
else if (...) { score = 15; }

lobby.drawer.setScore(prevScore + score);
```